about.html

```diff
@@ -0,0 +1,304 @@
+                                </li>
+                                <li>
+                                    <a href="##">Program</a>
+                                </li>
+                                <li>
+                                    <a href="##">Join</a>
+                                </li>
+                                <li>
+                                    <a href="##">Sponsor</a>
+                                </li>
+                                <li>
+                                    <a href="##">News</a>
+                                </li>
+                                <li>
+                                    <a href="##">Follow us</a>
+                                </li>
+                            </ul>
+                            <span class="closeButton" id="close_button_about">&times;</span>
+                        </div>
+                    </div>
+                </div>
+
+
+            </div>
+            <div class="row">
+
+                <div class="container">
+                    <div class="row">
+                        <div class="col-3 desk-div"></div>
+                        <div class="col-9">
+                            <h1 class="about_header">RR Conference.
+                                <br> <span class="about_titlespan">South Africa conference 2023</span>
+                            </h1>
+                        </div>
+                    </div>
+                    <div class="row">
+
+                        <div class="col-12 about_paragraph_second">
+
+                            <p>
+                            <p> In particular, this year's summit is our third since incession</p>
+                              It's necessary when you have a company larger than 50 people, 
+                              when you need to share vision and values with the whole company 
+                              the best way to do it is in a conference format. This is also a 
+                              great format for teaching and sharing learnings from projects and people management. 
+                            </p>
+                            
+                            <p class="about_paragraph_box">March 2023.03.24(FRI)<br>
+                                See you at the conference Venue</p>
+
+                        </div>
+                    </div> 
+                    <div class="row">
+                        <div class="col-12 ">
+                            <p class="about_buttom_paragraph">Other questions about RR-Conference? <br>
+                                <span class="organisation"><a href="">kayembejonathan5gmail</a> </span>
+                            </p>
+                        </div>
+                    </div>
+
+                </div>
+            </div>
+        </div>
+        <div>
+            <!-- ---------------------Middle section about page  -->
+            <div class="container-fluid about_middle_section">
```

Comment:
- [ ] Kindly consider increasing the margin between this section as the section before it to make it similar to the original design.